corehq/apps/hqadmin/views.py

```diff
@@ -392,24 +394,52 @@ def stop_pillow(request):
         return get_response("No pillow found with name '{}'".format(pillow_name))
 
 
-@require_superuser
-@require_GET
-def admin_restore(request, app_id=None):
-    full_username = request.GET.get('as', '')
-    if not full_username or '@' not in full_username:
-        return HttpResponseBadRequest('Please specify a user using ?as=user@domain')
-
-    username, domain = full_username.split('@')
-    if not domain.endswith(settings.HQ_ACCOUNT_ROOT):
-        full_username = format_username(username, domain)
-
-    user = CommCareUser.get_by_username(full_username)
-    if not user:
-        return HttpResponseNotFound('User %s not found.' % full_username)
-
-    overwrite_cache = request.GET.get('ignore_cache') == 'true'
-    return get_restore_response(user.domain, user, overwrite_cache=overwrite_cache, app_id=app_id,
-                                **get_restore_params(request))
+class AdminRestoreView(TemplateView):
+    template_name = 'hqadmin/admin_restore.html'
+
+    @use_bootstrap3
```

Comment:
I think this is going to break when https://github.com/dimagi/commcare-hq/pull/11958/commits/6c50008ef9ac23c08f00a43875825b21821d37a2 gets merged